.github/workflows/CI.yml

```diff
@@ -36,6 +36,14 @@ jobs:
       - uses: actions/checkout@v4
       - name: Update Rust
         run: rustup update stable && rustup default stable
+      - name: Install protoc (Linux/macOS)
+        if: runner.os != 'Windows'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
```

Comment:
Rather than manually installing protoc, we can probably make use of this https://github.com/marketplace/actions/setup-protoc